config/storybook/preview-head.html

```diff
@@ -3,3 +3,6 @@
   href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
   rel="stylesheet"
 />
+<script>
+  window.global = window;
```

Comment:
what is this for? :-O

Reply:
for Storybook to work correctly, without it throws an error: `Uncaught ReferenceError: global is not defined
    at ../../node_modules/.pnpm/jest-mock@27.5.1/node_modules/jest-mock/build/index.js `